fast_api_als/database/db_helper.py

```diff
@@ -288,32 +316,37 @@ def insert_customer_lead(self, uuid: str, email: str, phone: str, last_name: str
             'ttl': datetime.fromtimestamp(int(time.time())) + timedelta(days=constants.OEM_ITEM_TTL)
         }
         res = self.table.put_item(Item=item)
+        log_db_self_table(res, "put_item")
 
     def lead_exists(self, uuid: str, make: str, model: str):
         lead_exist = False
         if self.get_make_model_filter_status(make):
             res = self.table.query(
                 KeyConditionExpression=Key('pk').eq(f"{make}#{uuid}") & Key('sk').eq(f"{make}#{model}")
             )
+            log_db_self_table(res, "query")
             if len(res['Items']):
                 lead_exist = True
         else:
             res = self.table.query(
                 KeyConditionExpression=Key('pk').eq(f"{make}#{uuid}")
             )
+            log_db_self_table(res, "query")
             if len(res['Items']):
                 lead_exist = True
         return lead_exist
 
     def check_duplicate_lead(self, email: str, phone: str, last_name: str, make: str, model: str):
-        email_attached_leads = self.table.query(
+         = self.table.query(
```

Comment:
will this code execute correctly?

Reply:
Really sorry, instead of copying the name, I cut it by mistake. Didn't review it.

Reply:
This is happening in one more place/places.
Please review it.

I'm seeing you're not putting much effort.